dmutils/content_loader.py

```diff
@@ -494,6 +497,10 @@ def _optional_form_fields(self):
         else:
             return []
 
+    def inject_brief_questions_into_boolean_list_question(self, brief):
```

Comment:
Method names are exactly the same for several of the other method names (see `summary`, `get_question` or `get_error_messages`).

But I'll change it from `brief`, no problem.